### ISSUE TITLE
validation error for "git show -q"

### ISSUE THREAD

**Issue by ciscoqid** [librenms/librenms#14132]
### The problem

validate.php results in this on CentOS 7:

[FAIL]  Failed to fetch version from local git: fatal: unrecognized argument: -q

This is because git is at version 1.8.3.1 on this distribution and that option to git-show throws an error.  Testing in Rocky 8 with git  2.31.1 does not throw an error but the output is no different with or without -q (and -q is not listed as an option in the git-show man page).  I recommend removing -q from the argument list in this line:

$process = new Process(['git', 'show', '-q', '--pretty=%H|%ct'], base_path())

If it really buys something, then a version check needs to be done to conditionally include -q.

### Output of ./validate.php

```txt
====================================
Component | Version
--------- | -------
LibreNMS  | 22.7.0-537-gcc00ce1
DB Schema | 2022_07_19_081224_plugins_unique_index (243)
PHP       | 7.3.33
Python    | 3.6.8
MySQL     | 10.5.16-MariaDB
RRDTool   | 1.4.8
SNMP      | 5.7.2
====================================

[OK]    Composer Version: 2.3.10
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    
[OK]    Database schema correct
[OK]    MySQl and PHP time match
[OK]    rrd_dir is writable
[OK]    rrdtool version ok
[FAIL]  Failed to fetch version from local git: fatal: unrecognized argument: -q
[WARN]  Your local git contains modified files, this could prevent automatic updates.
        [FIX]: 
        You can fix this with ./scripts/github-remove
        Modified Files:
         html/.htaccess
```

**Comment by ciscoqid** [comment]
I wish that fixed it, but the --no-patch option is not available in the CentOS 7 git version (1.8.3.1).  The man page indicates that git diff-tree options are also accepted, and --quiet is one of those options (but -q does not work).  So it seems the right general fix is to replace -q with --quiet.

**Comment by murrant** [comment]
@ciscoqid feel free to send PR
